cogs/games.py

```diff
@@ -108,13 +128,50 @@ def __init__(self, bot):
 		self.config = ConfigTable(bot, 'trivia', ('guild_id', 'user_id'))
 		self.playing = set()
 
+		asyncio.run_coroutine_threadsafe(self.get_trivia_categories(), self.bot.loop)
+
+	async def get_trivia_categories(self):
+		global TRIVIA_CATEGORIES
```

Comment:
Wow, I had no clue that `ctx.cog` was a thing, that's super useful. 
I only had `TRIVIA_CATEGORIES` global since the converter and cog both need it, I've changed it to be the cog member `trivia_categories` now that it doesn't have to be global.